Prose/ProseLib/Sources/App/AppRoute.swift

```diff
@@ -9,7 +9,7 @@ import AuthenticationFeature
 import MainWindowFeature
 
 public enum AppRoute: Equatable {
-    case auth(AuthenticationState)
+    case auth(AuthRoute)
```

Comment:
Ok, I'll fix both of these.

> the AuthenticationFeature decides which route to show and not the AppScene

Can't `authenticationReducer` say `state = .basic` for example?

Reply:
Internally, sure:

```swift
public enum AuthenticationState: Equatable {
    var route = Route.basic(BasicAuthState())
}
```

Reply:
I mean, if the state is an `enum`, `authenticationReducer` can still change the case, with `state =`.

I thought you were saying otherwise

> the `AuthenticationFeature` decides which route to show and not the AppScene

(By the way, I'm just answering, I will fix this anyway)